libraries/standard/http/utest/http_utest.c

```diff
@@ -229,7 +228,11 @@ void test_Http_InitializeRequestHeaders_Happy_path()
                          HTTP_USER_AGENT_FIELD, HTTP_USER_AGENT_VALUE,
                          HTTP_HOST_FIELD, HTTP_TEST_HOST_VALUE,
                          HTTP_CONNECTION_FIELD, HTTP_CONNECTION_CLOSE_VALUE );
+    /* Make sure that the entire pre-existing data was printed to the buffer. */
     TEST_ASSERT_EQUAL( expectedHeaders.dataLen, numBytes );
```

Comment:
This test is not required anymore as the following 2 asserts represent successful printing of data to the buffer